ush/parsing_namelists_WW3.sh

```diff
@@ -75,252 +42,55 @@ WW3_namelists(){
-      -e "s/UNIPOINTS/${UNIPOINTS}/g" \
-      -e "s/GRIDLINE/${gline}/g" \
-      -e "s/ICELINE/$ICELINE/g" \
-      -e "s/CURRLINE/$CURRLINE/g" \
-      -e "s/WINDLINE/$WINDLINE/g" \
-      -e "s/ICEFLAG/$ICEFLAG/g" \
-      -e "s/CURRFLAG/$CURRFLAG/g" \
-      -e "s/WINDFLAG/$WINDFLAG/g" \
-      -e "s/RUN_BEG/$time_beg/g" \
-      -e "s/RUN_END/$time_end/g" \
-      -e "s/OUT_BEG/$time_beg_out/g" \
-      -e "s/OUT_END/$time_end/g" \
-      -e "s/DTFLD/ $DTFLD_WAV/g" \
-      -e "s/FLAGMASKCOMP/ $FLAGMASKCOMP/g" \
-      -e "s/FLAGMASKOUT/ $FLAGMASKOUT/g" \
-      -e "s/GOFILETYPE/ $GOFILETYPE/g" \
-      -e "s/POFILETYPE/ $POFILETYPE/g" \
-      -e "s/DTPNT/ $DTPNT_WAV/g" \
-      -e "/BUOY_FILE/r buoy.loc" \
-      -e "s/BUOY_FILE/DUMMY/g" \
-      -e "s/RST_BEG/$time_rst_ini/g" \
-      -e "s/RSTTYPE/$RSTTYPE_WAV/g" \
-      -e "s/RST_2_BEG/$time_rst2_ini/g" \
-      -e "s/DTRST/$DT_1_RST_WAV/g" \
-      -e "s/DT_2_RST/$DT_2_RST_WAV/g" \
-      -e "s/RST_END/$time_rst1_end/g" \
-      -e "s/RST_2_END/$time_rst2_end/g" \
-                                     ww3_multi.inp.tmpl | \
-  sed -n "/DUMMY/!p"               > ww3_multi.inp
-
-  rm -f ww3_multi.inp.tmpl buoy.loc
-
-  cat ww3_multi.inp
-
-else 
-  #ww3_shel 
-
-# Initialize inp file parameters
-  ICELINE='F F'
-  CURRLINE='F F'
-  WINDLINE='F F'
-
-  case ${WW3ATMINP} in
-    'YES' )
-      WINDLINE="T F";;
-    'CPL' )
-      WINDLINE="C F";;
-  esac
+  #set coupling to ice/current
+  WW3_ICE='F'
+  WW3_IC1='F'
+  WW3_IC5='F'
+  WW3_CUR='F'
 
   case ${WW3ICEINP} in
     'YES' )
-      ICELINE="T F";;
+      WW3_ICE="T";;
     'CPL' )
-      ICELINE="C F";;
+      WW3_ICE="C";;
   esac
 
   case ${WW3CURINP} in
     'YES' )
-      CURRLINE="T F";;
+      WW3_CUR="T";;
     'CPL' )
-      CURRLINE="C F";;
+      WW3_CUR="C";;
   esac
 
-  sed -e "s/IOSRV/${IOSRV}/g" \
-      -e "s/OUTPARS/${OUTPARS_WAV}/g" \
-      -e "s/ICELINE/$ICELINE/g" \
-      -e "s/CURRLINE/$CURRLINE/g" \
-      -e "s/WINDLINE/$WINDLINE/g" \
-      -e "s/RUN_BEG/$time_beg/g" \
-      -e "s/RUN_END/$time_end/g" \
-      -e "s/OUT_BEG/$time_beg_out/g" \
-      -e "s/OUT_END/$time_end/g" \
-      -e "s/DTFLD/ $DTFLD_WAV/g" \
-      -e "s/GOFILETYPE/ $GOFILETYPE/g" \
-      -e "s/POFILETYPE/ $POFILETYPE/g" \
-      -e "s/DTPNT/ $DTPNT_WAV/g" \
-      -e "s/DTPNT/ $DTPNT_WAV/g" \
-      -e "/BUOY_FILE/r buoy.loc" \
-      -e "s/BUOY_FILE/DUMMY/g" \
-      -e "s/RST_BEG/$time_rst_ini/g" \
-      -e "s/RSTTYPE/$RSTTYPE_WAV/g" \
-      -e "s/RST_2_BEG/$time_rst2_ini/g" \
-      -e "s/DTRST/$DT_1_RST_WAV/g" \
-      -e "s/DT_2_RST/$DT_2_RST_WAV/g" \
-      -e "s/RST_END/$time_rst1_end/g" \
-      -e "s/RST_2_END/$time_rst2_end/g" \
-                                     ww3_shel.inp.tmpl | \
-  sed -n "/DUMMY/!p"               > ww3_shel.inp
-
-  rm -f ww3_shel.inp.tmpl buoy.loc
-
-  cat ww3_shel.inp
-
-fi   
+  local WW3_OUTPARS=$OUTPARS_WAV 
+  local WW3_DTFLD=$DTFLD_WAV
+  local WW3_DTPNT=$DTPNT_WAV
```

Comment:
```suggestion
  local WW3_OUTPARS="${OUTPARS_WAV}"
  local WW3_DTFLD="${DTFLD_WAV}"
  local WW3_DTPNT="${DTPNT_WAV}"
```
Not sure why shellnorms did not pick up on this.